projects/adi_vna_8ch/common/adi_vna_8ch_bd.tcl

```diff
@@ -0,0 +1,347 @@
+ad_connect  spim_sdi_i  axi_spim/io1_i
+
+ad_connect  sys_cpu_clk  axi_spi_adl5960_1/ext_spi_clk
+ad_connect  spi_adl5960_1_csn_i  axi_spi_adl5960_1/ss_i
+ad_connect  spi_adl5960_1_csn_o  axi_spi_adl5960_1/ss_o
+ad_connect  spi_adl5960_1_clk_i  axi_spi_adl5960_1/sck_i
+ad_connect  spi_adl5960_1_clk_o  axi_spi_adl5960_1/sck_o
+ad_connect  spi_adl5960_1_sdo_i  axi_spi_adl5960_1/io0_i
+ad_connect  spi_adl5960_1_sdo_o  axi_spi_adl5960_1/io0_o
+ad_connect  spi_adl5960_1_sdi_i  axi_spi_adl5960_1/io1_i
+
+ad_connect  sys_cpu_clk  axi_spi_fpga_busf/ext_spi_clk
+ad_connect  spi_fpga_busf_csn_i  axi_spi_fpga_busf/ss_i
+ad_connect  spi_fpga_busf_csn_o  axi_spi_fpga_busf/ss_o
+ad_connect  spi_fpga_busf_clk_i  axi_spi_fpga_busf/sck_i
+ad_connect  spi_fpga_busf_clk_o  axi_spi_fpga_busf/sck_o
+ad_connect  spi_fpga_busf_sdo_i  axi_spi_fpga_busf/io0_i
+ad_connect  spi_fpga_busf_sdo_o  axi_spi_fpga_busf/io0_o
+ad_connect  spi_fpga_busf_sdi_i  axi_spi_fpga_busf/io1_i
+
+ad_connect  sys_cpu_clk  axi_spi_fmcdac/ext_spi_clk
+ad_connect  spi_fmcdac_csn_i  axi_spi_fmcdac/ss_i
+ad_connect  spi_fmcdac_csn_o  axi_spi_fmcdac/ss_o
+ad_connect  spi_fmcdac_clk_i  axi_spi_fmcdac/sck_i
+ad_connect  spi_fmcdac_clk_o  axi_spi_fmcdac/sck_o
+ad_connect  spi_fmcdac_sdo_i  axi_spi_fmcdac/io0_i
+ad_connect  spi_fmcdac_sdo_o  axi_spi_fmcdac/io0_o
+ad_connect  spi_fmcdac_sdi_i  axi_spi_fmcdac/io1_i
+
+ad_connect  sys_cpu_clk  axi_spi_ndac/ext_spi_clk
+ad_connect  ndac_spi_csn_i  axi_spi_ndac/ss_i
+ad_connect  ndac_spi_csn_o  axi_spi_ndac/ss_o
+ad_connect  ndac_spi_clk_i  axi_spi_ndac/sck_i
+ad_connect  ndac_spi_clk_o  axi_spi_ndac/sck_o
+ad_connect  ndac_spi_sdo_i  axi_spi_ndac/io0_i
+ad_connect  ndac_spi_sdo_o  axi_spi_ndac/io0_o
+ad_connect  ndac_spi_sdi_i  axi_spi_ndac/io1_i
+
+
+# interconnect (cpu)
+ad_cpu_interconnect 0x48000000 axi_fpga_bus0
+ad_cpu_interconnect 0x48100000 axi_fpga_bus1
+ad_cpu_interconnect 0x48200000 axi_spi_adl5960_1
+ad_cpu_interconnect 0x48300000 axi_spi_fmcdac
+ad_cpu_interconnect 0x48400000 axi_spi_fpga_busf
+ad_cpu_interconnect 0x48500000 axi_spim
+ad_cpu_interconnect 0x48600000 axi_spi_ndac
+
+ad_cpu_interconnect 0x44B60000 dac_jesd204_xcvr
+ad_cpu_interconnect 0x44B04000 dac_jesd204_transport
+ad_cpu_interconnect 0x44B90000 dac_jesd204_link
+ad_cpu_interconnect 0x7c420000 dac_dma
+
+# interrupts
+ad_cpu_interrupt ps-3  mb-12 axi_spi_fmcdac/ip2intc_irpt
+ad_cpu_interrupt ps-4  mb-11 dac_jesd204_link/irq
+ad_cpu_interrupt ps-5  mb-10 dac_dma/irq
+ad_cpu_interrupt ps-6  mb-10 axi_spi_ndac/ip2intc_irpt
+
+ad_cpu_interrupt ps-8  mb-7  axi_spi_adl5960_1/ip2intc_irpt
+ad_cpu_interrupt ps-9  mb-6  axi_spi_fpga_busf/ip2intc_irpt
+ad_cpu_interrupt ps-10 mb-5  axi_fpga_bus1/ip2intc_irpt
+ad_cpu_interrupt ps-11 mb-4  axi_fpga_bus0/ip2intc_irpt
+ad_cpu_interrupt ps-15 mb-14  axi_spim/ip2intc_irpt
+
+# interconnect mem
+
+ad_mem_hp3_interconnect $sys_dma_clk sys_ps8/S_AXI_HP3
+ad_mem_hp3_interconnect $sys_dma_clk dac_dma/m_src_axi
```

Comment:
Add a newline character at the end of the file, as now the red minus sign is appearing (you can see it if you go to the "Files changed" section)